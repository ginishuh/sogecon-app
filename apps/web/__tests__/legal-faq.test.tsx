import { render, screen } from '@testing-library/react';
import React from 'react';
import { describe, expect, it } from 'vitest';

import FAQPage from '../app/faq/page';
import PrivacyPage from '../app/privacy/page';
import TermsPage from '../app/terms/page';

describe('FAQ & Policy static pages', () => {
  it('renders FAQ page with grouped sections and support information', () => {
    const { asFragment } = render(<FAQPage />);
    expect(screen.getByRole('heading', { level: 1, name: '자주 묻는 질문' })).toBeInTheDocument();
    expect(screen.getByRole('heading', { name: '계정 및 접근' })).toBeInTheDocument();
    expect(screen.getByText('문서 버전: 2025-10-08')).toBeInTheDocument();
    expect(screen.getByRole('link', { name: '이용약관' })).toHaveAttribute('href', '/terms');
    expect(asFragment()).toMatchSnapshot();
  });

  it('renders privacy policy with sections and contact info', () => {
    const { asFragment } = render(<PrivacyPage />);
    expect(screen.getByRole('heading', { level: 1, name: '개인정보 처리방침' })).toBeInTheDocument();
    expect(screen.getByRole('heading', { name: '6. 개인정보 보호를 위한 기술적·관리적 대책' })).toBeInTheDocument();
    expect(screen.getByText('김서강 회장 (office@sogang-econ-alumni.kr)')).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });

  it('renders terms of service with clauses and effective date', () => {
    const { asFragment } = render(<TermsPage />);
    expect(screen.getByRole('heading', { level: 1, name: '이용약관' })).toBeInTheDocument();
    expect(screen.getByRole('heading', { name: '제8조 (이용계약 해지 및 제한)' })).toBeInTheDocument();
    expect(screen.getByText('문서 버전: 2025-10-08 · 시행일: 2025-10-15')).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
