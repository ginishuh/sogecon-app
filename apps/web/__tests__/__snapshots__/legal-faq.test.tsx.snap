// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FAQ & Policy static pages > renders FAQ page with grouped sections and support information 1`] = `
<DocumentFragment>
  <div
    class="space-y-12 rounded-3xl bg-white px-6 py-12 shadow-soft md:px-12"
  >
    <header
      class="space-y-3"
    >
      <p
        class="text-xs font-semibold uppercase tracking-widest text-neutral-muted"
      >
        Support
      </p>
      <h1
        class="font-heading text-3xl text-neutral-ink md:text-4xl"
      >
        자주 묻는 질문
      </h1>
      <p
        class="text-sm leading-6 text-neutral-muted"
      >
        웹 런치와 함께 달라지는 회원 경험을 문답 형식으로 정리했습니다. 빠르게 궁금증을 해결하고, 추가 문의는 사무국으로 남겨 주세요.
      </p>
    </header>
    <div
      class="grid gap-8 md:grid-cols-2"
    >
      <section
        aria-labelledby="access-title"
        class="space-y-4 rounded-2xl border border-neutral-border bg-white/70 p-6 shadow-sm"
      >
        <div
          class="space-y-2"
        >
          <h2
            class="font-heading text-xl text-brand-primary"
            id="access-title"
          >
            계정 및 접근
          </h2>
          <p
            class="text-sm text-neutral-muted"
          >
            로그인, 인증, 권한에 대한 핵심 질문을 정리했습니다.
          </p>
        </div>
        <dl
          aria-label="계정 및 접근 질문 목록"
          class="space-y-5"
        >
          <div
            class="rounded-xl bg-brand-surface/60 p-4"
          >
            <dt
              class="text-sm font-semibold text-neutral-ink"
            >
              웹 런치 이후 로그인 방식은 어떻게 바뀌나요?
            </dt>
            <dd
              class="mt-2 text-sm leading-6 text-neutral-muted"
            >
              학번(ID)과 초기 비밀번호 체계는 유지되지만, 최초 로그인 시 본인 확인 후 반드시 새로운 비밀번호를 설정해야 합니다.
              <br />
              비활성 상태인 회원은 
              <strong>
                회원활성화
              </strong>
               메뉴에서 본인 인증(생년월일 또는 등록 이메일) 절차를 진행해 주세요.
            </dd>
          </div>
          <div
            class="rounded-xl bg-brand-surface/60 p-4"
          >
            <dt
              class="text-sm font-semibold text-neutral-ink"
            >
              동문이 아니라도 디렉터리를 볼 수 있나요?
            </dt>
            <dd
              class="mt-2 text-sm leading-6 text-neutral-muted"
            >
              아니요. 디렉터리와 회원 프로필은 활성화된 총원우회 회원만 열람할 수 있습니다. 로그인하지 않은 사용자는 기본 안내 페이지만 볼 수 있으며, 본문 접근 시 로그인 화면으로 리다이렉트됩니다.
            </dd>
          </div>
          <div
            class="rounded-xl bg-brand-surface/60 p-4"
          >
            <dt
              class="text-sm font-semibold text-neutral-ink"
            >
              로그인에 실패했을 때 무엇을 확인해야 하나요?
            </dt>
            <dd
              class="mt-2 text-sm leading-6 text-neutral-muted"
            >
              비밀번호 재설정 링크는 
              <em>
                가입 시 등록한 이메일
              </em>
              로 발송됩니다. 메일을 받지 못했다면 스팸함을 확인하거나 
              <a
                href="mailto:office@sogang-econ-alumni.kr"
              >
                office@sogang-econ-alumni.kr
              </a>
              로 문의해 주세요. 5회 이상 연속 실패 시 보안을 위해 10분간 로그인이 제한됩니다.
            </dd>
          </div>
        </dl>
      </section>
      <section
        aria-labelledby="features-title"
        class="space-y-4 rounded-2xl border border-neutral-border bg-white/70 p-6 shadow-sm"
      >
        <div
          class="space-y-2"
        >
          <h2
            class="font-heading text-xl text-brand-primary"
            id="features-title"
          >
            서비스 기능
          </h2>
          <p
            class="text-sm text-neutral-muted"
          >
            홈 화면, 공지, 행사, 디렉터리에서 자주 묻는 질문을 모았습니다.
          </p>
        </div>
        <dl
          aria-label="서비스 기능 질문 목록"
          class="space-y-5"
        >
          <div
            class="rounded-xl bg-brand-surface/60 p-4"
          >
            <dt
              class="text-sm font-semibold text-neutral-ink"
            >
              홈 화면에서 어떤 정보를 빠르게 확인할 수 있나요?
            </dt>
            <dd
              class="mt-2 text-sm leading-6 text-neutral-muted"
            >
              히어로 영역에서 다음 행사를 확인하고, 공지·FAQ·디렉터리 바로가기 카드를 통해 주요 기능으로 이동할 수 있습니다. 모바일에서도 동일한 레이아웃을 제공하며, 즐겨찾기 버튼을 이용하면 홈 화면에 바로가기 아이콘을 추가할 수 있습니다.
            </dd>
          </div>
          <div
            class="rounded-xl bg-brand-surface/60 p-4"
          >
            <dt
              class="text-sm font-semibold text-neutral-ink"
            >
              디렉터리 검색은 어떤 방식으로 작동하나요?
            </dt>
            <dd
              class="mt-2 text-sm leading-6 text-neutral-muted"
            >
              이름·이메일 키워드 검색과 함께 기수, 업종, 지역, 회사, 전공 필터를 지원합니다. 필터를 변경하면 350ms 디바운스를 거친 후 자동으로 URL이 업데이트돼, 공유 링크에 현재 검색 조건이 반영됩니다.
            </dd>
          </div>
          <div
            class="rounded-xl bg-brand-surface/60 p-4"
          >
            <dt
              class="text-sm font-semibold text-neutral-ink"
            >
              행사와 공지는 어디서 관리하나요?
            </dt>
            <dd
              class="mt-2 text-sm leading-6 text-neutral-muted"
            >
              일반 회원은 홈과 
              <a
                href="/events"
              >
                행사 페이지
              </a>
              에서 최신 일정을 확인할 수 있습니다. 운영진은 관리자 콘솔에 로그인해 공지/행사 등록, 푸시 발송, 통계 확인을 진행합니다.
            </dd>
          </div>
        </dl>
      </section>
      <section
        aria-labelledby="policy-title"
        class="space-y-4 rounded-2xl border border-neutral-border bg-white/70 p-6 shadow-sm"
      >
        <div
          class="space-y-2"
        >
          <h2
            class="font-heading text-xl text-brand-primary"
            id="policy-title"
          >
            정책 및 지원
          </h2>
          <p
            class="text-sm text-neutral-muted"
          >
            개인정보 처리와 문의 채널을 안내합니다.
          </p>
        </div>
        <dl
          aria-label="정책 및 지원 질문 목록"
          class="space-y-5"
        >
          <div
            class="rounded-xl bg-brand-surface/60 p-4"
          >
            <dt
              class="text-sm font-semibold text-neutral-ink"
            >
              개인정보는 어디에 저장되며 어떻게 보호되나요?
            </dt>
            <dd
              class="mt-2 text-sm leading-6 text-neutral-muted"
            >
              회원 정보는 암호화된 데이터베이스에 저장되며, 관리자 접근은 역할 기반 권한(RBAC)으로 제한됩니다. 열람 기록은 1년간 감사 로그에 보관되고, 404/410 응답을 반환하는 푸시 구독 정보는 즉시 폐기됩니다. 자세한 내용은 
              <a
                href="/privacy"
              >
                개인정보 처리방침
              </a>
              을 참고하세요.
            </dd>
          </div>
          <div
            class="rounded-xl bg-brand-surface/60 p-4"
          >
            <dt
              class="text-sm font-semibold text-neutral-ink"
            >
              정책 문서는 어디에서 확인할 수 있나요?
            </dt>
            <dd
              class="mt-2 text-sm leading-6 text-neutral-muted"
            >
              본 페이지와 함께 
              <a
                href="/terms"
              >
                이용약관
              </a>
              , 
              <a
                href="/privacy"
              >
                개인정보 처리방침
              </a>
              을 제공합니다. 문서 버전은 꼬리말에 명시되며, 개정 시 등록 이메일로 사전 안내를 드립니다.
            </dd>
          </div>
          <div
            class="rounded-xl bg-brand-surface/60 p-4"
          >
            <dt
              class="text-sm font-semibold text-neutral-ink"
            >
              문의는 어떤 채널을 이용하면 되나요?
            </dt>
            <dd
              class="mt-2 text-sm leading-6 text-neutral-muted"
            >
              사무국 대표 메일(
              <a
                href="mailto:office@sogang-econ-alumni.kr"
              >
                office@sogang-econ-alumni.kr
              </a>
              ) 또는 대표 전화(02-715-1234)로 연락해 주세요. 평일 10:00-17:00에 응대하며, 주말 문의는 다음 영업일에 순차 처리합니다.
            </dd>
          </div>
        </dl>
      </section>
    </div>
    <section
      aria-labelledby="faq-support"
      class="rounded-2xl bg-brand-surface px-6 py-6 text-sm text-neutral-muted md:py-8"
    >
      <h2
        class="font-heading text-lg text-brand-primary"
        id="faq-support"
      >
        추가로 궁금한 점이 있다면
      </h2>
      <p
        class="mt-2"
      >
        사무국 대표 메일 
        <a
          class="underline decoration-brand-accent decoration-2 underline-offset-4"
          href="mailto:office@sogang-econ-alumni.kr"
        >
          office@sogang-econ-alumni.kr
        </a>
         또는 대표 번호 02-715-1234(ARS 2번)로 연락해 주세요. 평균 응답 시간은 영업일 기준 24시간 이내입니다.
      </p>
    </section>
    <p
      class="text-xs uppercase tracking-widest text-neutral-muted"
    >
      문서 버전: 2025-10-08
    </p>
  </div>
</DocumentFragment>
`;

exports[`FAQ & Policy static pages > renders privacy policy with sections and contact info 1`] = `
<DocumentFragment>
  <div
    class="space-y-12 rounded-3xl bg-white px-6 py-12 shadow-soft md:px-12"
  >
    <header
      class="space-y-3"
    >
      <p
        class="text-xs font-semibold uppercase tracking-widest text-neutral-muted"
      >
        Policy
      </p>
      <h1
        class="font-heading text-3xl text-neutral-ink md:text-4xl"
      >
        개인정보 처리방침
      </h1>
      <p
        class="text-sm leading-6 text-neutral-muted"
      >
        서강대 경제대학원 총원우회는 『개인정보 보호법』, 『정보통신망 이용촉진 및 정보보호 등에 관한 법률』 등 관련 법령을 준수하며 회원 정보를 보호합니다. 아래 항목을 확인하시고, 문의 사항은 언제든지 개인정보 보호책임자에게 전달해 주세요.
      </p>
    </header>
    <div
      class="space-y-10"
    >
      <section
        aria-labelledby="collection-title"
        class="space-y-4 rounded-2xl border border-neutral-border bg-white/70 p-6 shadow-sm"
      >
        <div
          class="space-y-2"
        >
          <h2
            class="font-heading text-xl text-brand-primary"
            id="collection-title"
          >
            1. 수집하는 개인정보 항목
          </h2>
          <p
            class="text-sm text-neutral-muted"
          >
            회원 가입 및 서비스 이용 과정에서 필요한 최소한의 개인정보만 수집합니다.
          </p>
        </div>
        <div
          class="space-y-3 text-sm leading-6 text-neutral-muted"
        >
          <ul
            class="list-disc space-y-2 pl-5"
          >
            <li>
              <strong>
                필수 항목
              </strong>
               · 학번(ID), 이름, 기수, 생년월일(본인확인용), 이메일, 휴대전화, 가입 승인 기록
            </li>
            <li>
              <strong>
                선택 항목
              </strong>
               · 전공, 직장명, 직함, 업종, 주소(개인/직장), 프로필 이미지
            </li>
            <li>
              서비스 이용 기록(로그인 일시, 접속 IP, 기기 정보), 문의 및 상담 기록, 푸시 구독 토큰
            </li>
          </ul>
        </div>
      </section>
      <section
        aria-labelledby="purpose-title"
        class="space-y-4 rounded-2xl border border-neutral-border bg-white/70 p-6 shadow-sm"
      >
        <div
          class="space-y-2"
        >
          <h2
            class="font-heading text-xl text-brand-primary"
            id="purpose-title"
          >
            2. 개인정보 이용 목적
          </h2>
          <p
            class="text-sm text-neutral-muted"
          >
            명시된 목적 범위 내에서만 개인정보를 이용하며, 목적 변경 시 별도 동의를 받습니다.
          </p>
        </div>
        <div
          class="space-y-3 text-sm leading-6 text-neutral-muted"
        >
          <ol
            class="list-decimal space-y-2 pl-5"
          >
            <li>
              회원 식별 및 로그인, 계정 활성화·재설정 등 서비스 기본 기능 제공
            </li>
            <li>
              행사/공지/디렉터리 운영을 위한 회원 정보 검증과 연락
            </li>
            <li>
              장학사업, 프로그램 안내, 총회 공지 등 커뮤니케이션 발송
            </li>
            <li>
              서비스 안정성 확보, 보안 로그 분석, 이용 통계 작성
            </li>
          </ol>
        </div>
      </section>
      <section
        aria-labelledby="retention-title"
        class="space-y-4 rounded-2xl border border-neutral-border bg-white/70 p-6 shadow-sm"
      >
        <div
          class="space-y-2"
        >
          <h2
            class="font-heading text-xl text-brand-primary"
            id="retention-title"
          >
            3. 보유 및 이용 기간
          </h2>
          <p
            class="text-sm text-neutral-muted"
          >
            회원 탈퇴 후 6개월 보관 후 파기하며, 법령이 정한 기간은 별도 보관합니다.
          </p>
        </div>
        <div
          class="space-y-3 text-sm leading-6 text-neutral-muted"
        >
          <ul
            class="list-disc space-y-2 pl-5"
          >
            <li>
              회원 DB: 탈퇴 시 지체 없이 파기, 단 분쟁 대응을 위해 6개월간 최소 정보(이름, 연락처) 보관
            </li>
            <li>
              로그 기록: 통신비밀보호법에 따라 12개월 보관 후 파기
            </li>
            <li>
              장학·재정 관련 증빙: 국세기본법 등 관련 법령에 따라 최대 5년 보관
            </li>
          </ul>
        </div>
      </section>
      <section
        aria-labelledby="third-party-title"
        class="space-y-4 rounded-2xl border border-neutral-border bg-white/70 p-6 shadow-sm"
      >
        <div
          class="space-y-2"
        >
          <h2
            class="font-heading text-xl text-brand-primary"
            id="third-party-title"
          >
            4. 제3자 제공 및 위탁
          </h2>
          <p
            class="text-sm text-neutral-muted"
          >
            법령 근거가 있거나 회원 동의를 받은 경우에 한해 외부에 제공합니다.
          </p>
        </div>
        <div
          class="space-y-3 text-sm leading-6 text-neutral-muted"
        >
          <p>
            총원우회는 원칙적으로 회원 개인정보를 외부에 제공하지 않습니다. 다만 다음의 경우에는 예외적으로 제공할 수 있으며, 제공 시 사전 고지 및 동의를 진행합니다.
          </p>
          <ul
            class="mt-3 list-disc space-y-2 pl-5"
          >
            <li>
              회원 동의가 있는 경우
            </li>
            <li>
              법령에 따라 수사기관 등으로부터 적법한 절차에 따라 제공 요청이 있는 경우
            </li>
            <li>
              클라우드 인프라, 문자 발송 등 서비스 운영에 필요한 위탁이 필요한 경우
            </li>
          </ul>
          <p
            class="mt-3"
          >
            현재 위탁 현황은 AWS(인프라 운영), Twilio(알림 발송)이며, 위탁 계약 시 안전성 확보를 위한 조치를 명시합니다.
          </p>
        </div>
      </section>
      <section
        aria-labelledby="rights-title"
        class="space-y-4 rounded-2xl border border-neutral-border bg-white/70 p-6 shadow-sm"
      >
        <div
          class="space-y-2"
        >
          <h2
            class="font-heading text-xl text-brand-primary"
            id="rights-title"
          >
            5. 이용자 권리와 행사 방법
          </h2>
          <p
            class="text-sm text-neutral-muted"
          >
            언제든지 개인정보 열람, 정정, 삭제, 처리정지를 요청할 수 있습니다.
          </p>
        </div>
        <div
          class="space-y-3 text-sm leading-6 text-neutral-muted"
        >
          <ul
            class="list-disc space-y-2 pl-5"
          >
            <li>
              회원은 
              <strong>
                마이페이지 &gt; 개인정보 수정
              </strong>
              에서 직접 정보를 열람·수정할 수 있습니다.
            </li>
            <li>
              삭제 또는 처리정지를 원할 경우 사무국 대표 이메일(office@sogang-econ-alumni.kr)로 신청해 주세요. 접수일로부터 7일 이내 처리 결과를 안내합니다.
            </li>
            <li>
              대리인을 통한 요청은 위임장과 본인 신분증 사본을 제출해야 하며, 30일 이내 답변드립니다.
            </li>
          </ul>
        </div>
      </section>
      <section
        aria-labelledby="security-title"
        class="space-y-4 rounded-2xl border border-neutral-border bg-white/70 p-6 shadow-sm"
      >
        <div
          class="space-y-2"
        >
          <h2
            class="font-heading text-xl text-brand-primary"
            id="security-title"
          >
            6. 개인정보 보호를 위한 기술적·관리적 대책
          </h2>
          <p
            class="text-sm text-neutral-muted"
          >
            암호화, 접근 통제, 로그 모니터링 등 다양한 보안 조치를 적용합니다.
          </p>
        </div>
        <div
          class="space-y-3 text-sm leading-6 text-neutral-muted"
        >
          <ul
            class="list-disc space-y-2 pl-5"
          >
            <li>
              비밀번호·연락처 등 주요 정보는 암호화 저장하며, TLS로 전송 구간을 보호합니다.
            </li>
            <li>
              관리자 계정은 다중 인증(MFA)을 필수로 적용하고, 역할 기반 권한(RBAC)을 운영합니다.
            </li>
            <li>
              접근 로그는 1년간 보관하며, 월 1회 이상 이상 징후를 점검합니다.
            </li>
            <li>
              연 1회 이상 보안 진단과 교육을 실시하고, 결과를 요약해 정기총회에 보고합니다.
            </li>
          </ul>
        </div>
      </section>
      <section
        aria-labelledby="contact-title"
        class="space-y-4 rounded-2xl border border-neutral-border bg-white/70 p-6 shadow-sm"
      >
        <div
          class="space-y-2"
        >
          <h2
            class="font-heading text-xl text-brand-primary"
            id="contact-title"
          >
            7. 개인정보 보호책임자
          </h2>
          <p
            class="text-sm text-neutral-muted"
          >
            문의 및 민원 처리를 위한 책임자 정보를 제공합니다.
          </p>
        </div>
        <div
          class="space-y-3 text-sm leading-6 text-neutral-muted"
        >
          <dl
            class="space-y-2"
          >
            <div>
              <dt
                class="font-semibold text-neutral-ink"
              >
                개인정보 보호책임자
              </dt>
              <dd
                class="text-neutral-muted"
              >
                김서강 회장 (office@sogang-econ-alumni.kr)
              </dd>
            </div>
            <div>
              <dt
                class="font-semibold text-neutral-ink"
              >
                개인정보 보호담당자
              </dt>
              <dd
                class="text-neutral-muted"
              >
                박은영 사무국장 (privacy@sogang-econ-alumni.kr, 02-715-1234)
              </dd>
            </div>
          </dl>
        </div>
      </section>
    </div>
    <footer
      class="rounded-2xl bg-brand-surface px-6 py-6 text-xs uppercase tracking-widest text-neutral-muted md:py-8"
    >
      <p>
        문서 버전: 2025-10-08 · 시행일: 2025-10-15
      </p>
      <p
        class="mt-2 text-neutral-muted/80"
      >
        시행일: 2025-10-15 · 개정일: 2025-10-08
      </p>
    </footer>
  </div>
</DocumentFragment>
`;

exports[`FAQ & Policy static pages > renders terms of service with clauses and effective date 1`] = `
<DocumentFragment>
  <div
    class="space-y-12 rounded-3xl bg-white px-6 py-12 shadow-soft md:px-12"
  >
    <header
      class="space-y-3"
    >
      <p
        class="text-xs font-semibold uppercase tracking-widest text-neutral-muted"
      >
        Policy
      </p>
      <h1
        class="font-heading text-3xl text-neutral-ink md:text-4xl"
      >
        이용약관
      </h1>
      <p
        class="text-sm leading-6 text-neutral-muted"
      >
        서비스 이용 전에 아래 약관을 확인해 주세요. 총원우회는 회원에게 공정하고 안전한 서비스를 제공하기 위해 약관을 투명하게 운영합니다.
      </p>
    </header>
    <div
      class="space-y-10"
    >
      <section
        aria-labelledby="purpose-title"
        class="space-y-4 rounded-2xl border border-neutral-border bg-white/70 p-6 shadow-sm"
      >
        <h2
          class="font-heading text-xl text-brand-primary"
          id="purpose-title"
        >
          제1조 (목적)
        </h2>
        <div
          class="space-y-3 text-sm leading-6 text-neutral-muted"
        >
          <p>
            본 약관은 서강대학교 경제대학원 총원우회(이하 "총원우회")가 제공하는 웹 서비스(이하 "서비스")의 이용과 관련하여 총원우회와 회원 간의 권리, 의무 및 책임 사항을 규정함을 목적으로 합니다.
          </p>
        </div>
      </section>
      <section
        aria-labelledby="definitions-title"
        class="space-y-4 rounded-2xl border border-neutral-border bg-white/70 p-6 shadow-sm"
      >
        <h2
          class="font-heading text-xl text-brand-primary"
          id="definitions-title"
        >
          제2조 (용어의 정의)
        </h2>
        <div
          class="space-y-3 text-sm leading-6 text-neutral-muted"
        >
          <ul
            class="list-disc space-y-2 pl-5"
          >
            <li>
              "회원"이라 함은 본 약관에 동의하고 서비스를 이용하는 서강대 경제대학원 총원우회 구성원을 말합니다.
            </li>
            <li>
              "ID"는 회원 식별과 서비스 이용을 위해 회원이 부여받은 학번을 의미합니다.
            </li>
            <li>
              "비밀번호"는 회원의 개인정보 보호와 인증을 위하여 회원 본인이 설정한 문자와 숫자의 조합을 말합니다.
            </li>
            <li>
              "콘텐츠"는 총원우회가 서비스 상에 게시하거나 회원이 업로드하는 모든 자료를 의미합니다.
            </li>
          </ul>
        </div>
      </section>
      <section
        aria-labelledby="membership-title"
        class="space-y-4 rounded-2xl border border-neutral-border bg-white/70 p-6 shadow-sm"
      >
        <h2
          class="font-heading text-xl text-brand-primary"
          id="membership-title"
        >
          제3조 (회원 가입 및 자격)
        </h2>
        <div
          class="space-y-3 text-sm leading-6 text-neutral-muted"
        >
          <ol
            class="list-decimal space-y-2 pl-5"
          >
            <li>
              회원 가입은 총원우회 정회원으로 등록된 경우에 한하여 사용할 수 있습니다.
            </li>
            <li>
              회원은 최초 로그인 후 비밀번호를 변경해야 하며, 정보를 최신 상태로 유지할 의무가 있습니다.
            </li>
            <li>
              타인의 정보를 도용하거나 허위 정보를 입력한 경우, 서비스 이용이 제한될 수 있습니다.
            </li>
          </ol>
        </div>
      </section>
      <section
        aria-labelledby="obligations-title"
        class="space-y-4 rounded-2xl border border-neutral-border bg-white/70 p-6 shadow-sm"
      >
        <h2
          class="font-heading text-xl text-brand-primary"
          id="obligations-title"
        >
          제4조 (회원의 의무)
        </h2>
        <div
          class="space-y-3 text-sm leading-6 text-neutral-muted"
        >
          <ol
            class="list-decimal space-y-2 pl-5"
          >
            <li>
              회원은 본 약관과 관련 법령, 총원우회가 정한 서비스 이용 안내를 준수해야 합니다.
            </li>
            <li>
              회원은 자신의 계정 정보를 제3자에게 공유하거나 대여할 수 없습니다.
            </li>
            <li>
              회원은 서비스 이용 중 습득한 다른 회원의 개인정보를 본래 목적 외로 사용하거나 제3자에게 제공할 수 없습니다.
            </li>
            <li>
              게시물 작성 시 저작권을 침해하지 않으며, 허위 사실 또는 명예를 훼손하는 내용을 게시할 수 없습니다.
            </li>
          </ol>
        </div>
      </section>
      <section
        aria-labelledby="services-title"
        class="space-y-4 rounded-2xl border border-neutral-border bg-white/70 p-6 shadow-sm"
      >
        <h2
          class="font-heading text-xl text-brand-primary"
          id="services-title"
        >
          제5조 (서비스의 제공 및 변경)
        </h2>
        <div
          class="space-y-3 text-sm leading-6 text-neutral-muted"
        >
          <p>
            총원우회는 다음과 같은 서비스를 제공합니다.
          </p>
          <ul
            class="mt-2 list-disc space-y-2 pl-5"
          >
            <li>
              공지, 행사, 디렉터리, 소개 등 총원우회 관련 정보 제공
            </li>
            <li>
              회원 간 교류를 위한 커뮤니티, 알림 서비스
            </li>
            <li>
              총원우회 운영과 관련한 각종 신청 및 통계 기능
            </li>
          </ul>
          <p
            class="mt-3"
          >
            총원우회는 필요한 경우 서비스의 내용을 변경하거나 중단할 수 있으며, 변경 사항은 사전에 회원에게 공지합니다.
          </p>
        </div>
      </section>
      <section
        aria-labelledby="suspension-title"
        class="space-y-4 rounded-2xl border border-neutral-border bg-white/70 p-6 shadow-sm"
      >
        <h2
          class="font-heading text-xl text-brand-primary"
          id="suspension-title"
        >
          제6조 (서비스의 중단)
        </h2>
        <div
          class="space-y-3 text-sm leading-6 text-neutral-muted"
        >
          <ul
            class="list-disc space-y-2 pl-5"
          >
            <li>
              시스템 점검, 교체, 고장 등 부득이한 사유가 발생한 경우 서비스 제공이 일시 중단될 수 있습니다.
            </li>
            <li>
              천재지변, 정전, 통신 장애 등 불가항력적 사유로 인한 서비스 중단 시 총원우회는 지체 없이 회원에게 공지합니다.
            </li>
          </ul>
        </div>
      </section>
      <section
        aria-labelledby="intellectual-property-title"
        class="space-y-4 rounded-2xl border border-neutral-border bg-white/70 p-6 shadow-sm"
      >
        <h2
          class="font-heading text-xl text-brand-primary"
          id="intellectual-property-title"
        >
          제7조 (저작권의 귀속 및 이용 제한)
        </h2>
        <div
          class="space-y-3 text-sm leading-6 text-neutral-muted"
        >
          <ol
            class="list-decimal space-y-2 pl-5"
          >
            <li>
              서비스와 관련된 저작권 및 지식재산권은 총원우회에 귀속합니다.
            </li>
            <li>
              회원이 서비스 내에 게시한 콘텐츠의 저작권은 회원에게 있으며, 총원우회는 서비스 운영을 위해 비독점적으로 이를 이용할 수 있습니다.
            </li>
            <li>
              회원은 서비스 이용으로 얻은 정보를 총원우회의 사전 승인 없이 복제, 전송, 배포, 출판, 방송 등 기타 방법으로 이용하거나 제3자에게 제공할 수 없습니다.
            </li>
          </ol>
        </div>
      </section>
      <section
        aria-labelledby="termination-title"
        class="space-y-4 rounded-2xl border border-neutral-border bg-white/70 p-6 shadow-sm"
      >
        <h2
          class="font-heading text-xl text-brand-primary"
          id="termination-title"
        >
          제8조 (이용계약 해지 및 제한)
        </h2>
        <div
          class="space-y-3 text-sm leading-6 text-neutral-muted"
        >
          <ol
            class="list-decimal space-y-2 pl-5"
          >
            <li>
              회원은 언제든지 마이페이지에서 회원 탈퇴를 신청할 수 있으며, 총원우회는 관련 법령이 정한 범위 내에서 개인정보를 보유/파기합니다.
            </li>
            <li>
              회원이 약관을 위반하거나 아래 행위를 하는 경우 총원우회는 사전 통지 후 서비스 이용을 제한하거나 해지할 수 있습니다.
              <ul
                class="mt-2 list-disc space-y-1 pl-5"
              >
                <li>
                  타인의 정보를 도용하거나 부정하게 사용하는 행위
                </li>
                <li>
                  서비스 운영을 고의로 방해하는 행위
                </li>
                <li>
                  법령 또는 공공질서에 위반되는 콘텐츠 게시
                </li>
              </ul>
            </li>
          </ol>
        </div>
      </section>
      <section
        aria-labelledby="liability-title"
        class="space-y-4 rounded-2xl border border-neutral-border bg-white/70 p-6 shadow-sm"
      >
        <h2
          class="font-heading text-xl text-brand-primary"
          id="liability-title"
        >
          제9조 (면책 조항)
        </h2>
        <div
          class="space-y-3 text-sm leading-6 text-neutral-muted"
        >
          <ul
            class="list-disc space-y-2 pl-5"
          >
            <li>
              총원우회는 천재지변, 전쟁, 기간통신사업자의 서비스 중단 등 불가항력으로 인한 손해에 대해서는 책임을 부담하지 않습니다.
            </li>
            <li>
              회원의 귀책사유로 인한 서비스 이용 장애에 대해서는 총원우회가 책임을 지지 않습니다.
            </li>
            <li>
              회원이 서비스 내에서 제공한 정보 및 자료의 정확성에 대해 총원우회는 보증하지 않습니다.
            </li>
          </ul>
        </div>
      </section>
      <section
        aria-labelledby="jurisdiction-title"
        class="space-y-4 rounded-2xl border border-neutral-border bg-white/70 p-6 shadow-sm"
      >
        <h2
          class="font-heading text-xl text-brand-primary"
          id="jurisdiction-title"
        >
          제10조 (준거법 및 관할)
        </h2>
        <div
          class="space-y-3 text-sm leading-6 text-neutral-muted"
        >
          <p>
            본 약관과 관련하여 분쟁이 발생한 경우 대한민국 법을 준거법으로 하며, 분쟁으로 소송이 제기될 경우 민사소송법상의 관할 법원을 따릅니다.
          </p>
        </div>
      </section>
      <section
        aria-labelledby="appendix-title"
        class="space-y-4 rounded-2xl border border-neutral-border bg-white/70 p-6 shadow-sm"
      >
        <h2
          class="font-heading text-xl text-brand-primary"
          id="appendix-title"
        >
          부칙
        </h2>
        <div
          class="space-y-3 text-sm leading-6 text-neutral-muted"
        >
          <ul
            class="list-disc space-y-2 pl-5"
          >
            <li>
              본 약관은 2025년 10월 15일부터 시행됩니다.
            </li>
            <li>
              총원우회는 필요 시 약관을 개정할 수 있으며, 개정 시 최소 7일 전에 공지합니다.
            </li>
          </ul>
        </div>
      </section>
    </div>
    <footer
      class="rounded-2xl bg-brand-surface px-6 py-6 text-xs uppercase tracking-widest text-neutral-muted md:py-8"
    >
      <p>
        문서 버전: 2025-10-08 · 시행일: 2025-10-15
      </p>
      <p
        class="mt-2 text-neutral-muted/80"
      >
        시행일: 2025-10-15 · 개정일: 2025-10-08
      </p>
    </footer>
  </div>
</DocumentFragment>
`;
